### DISCUSSION TITLE
Configuration of doosan m1013 for curobo

### DISCUSSION THREAD

**Discussion by LuCarpentier83** [NVlabs/curobo#272] [Demos]
Hey,

We are sharing our work about the configuration of Doosan m1013 for curobo. Feel free to use it and tell us if you find any mistakes `;)`

  

https://github.com/NVlabs/curobo/assets/169827638/b08fd53f-a085-494a-a92f-67ff249fdff3

https://github.com/NVlabs/curobo/assets/169827638/bdbb36fb-ee24-47dc-b98d-fb46991d4849

https://github.com/NVlabs/curobo/assets/169827638/60ec153a-e56b-4e54-813e-47517bbbb7b6

Link to the GitHub repository : https://github.com/Lab-CORO/curobo_doosan/tree/master/src/m1013

**Comment by balakumar-s** [comment]
Thanks!
